docs/GETTING_STARTED_GUIDE.md

```diff
@@ -22,6 +22,14 @@ First and foremost, the metrics service requires a Kubernetes cluster that align
 | --------- |
 | 1.17-1.19 |
 
+## Dell EMC CSI Driver for PowerFlex
+
+This porject captures telemetry data about Kubernetes storage usage and performance which are provisioned through Dell EMC CSI (Container Storage Interface) Driver for PowerFlex. The metrics service requires CSI Driver for PowerFlex is deployed in Kubenetes cluster.
```

Comment:
"The metrics service requires CSI Driver for PowerFlex is deployed in Kubenetes cluster." -> "The metrics service requires that the CSI Driver for PowerFlex is deployed in the Kubernetes cluster."

Reply:
Good catch. It should spell out "CSI Driver for Dell EMC PowerFlex" as all the Driver names have the format "CSI Driver for Dell EMC <Platform>"